scripts/test-truncating-of-client-hello.py

```diff
@@ -65,17 +40,43 @@ def main():
 
     if args:
         run_only = set(args)
-    else:
-        run_only = None
 
-    conversations = {}
+    return host, port, run_only, run_exclude
+
+def help_msg():
+    print("Usage: <script-name> [-h hostname] [-p port] [[probe-name] ...]")
+    print(" -h hostname    name of the host to run the test against")
+    print("                localhost by default")
+    print(" -p port        port number to use for connection, 4433 by default")
+    print(" probe-name     if present, will run only the probes with given")
+    print("                names and not all of them, e.g \"sanity\"")
+    print(" -e probe-name  exclude the probe from the list of the ones run")
+    print("                may be specified multiple times")
+    print(" --help         this message")
+
+class Conversation():
+    def __init__(self, name):
+        self.name = name
+
+    name = None
+    root_node = None
+    passed = None
```

Comment:
You have right it doesn't make sense in this situation. I have updated it to instance variables.